### ISSUE TITLE
Can't Publish Site

### ISSUE THREAD

**Issue by jaythanelam** [st4ple/solid-jekyll#59]
I'm having issues getting styles to load. I've followed your directions regarding setting up the url, and baseurl but nothing seems to be working.

Here's the site: https://jaythanelam.github.io/mtm/

Please help!

**Comment by jaythanelam** [comment]
I have my config file set like this:

baseurl: "/mtm"
url: "https://jaythanelam.github.io"

but none of the css or js files seem to be loading. any idea what i'm doing wrong?

**Comment by jaythanelam** [comment]
I have tried all manner of changes to the baserul and the url, and nothing seems to be working after I push everything to github. I can get this work locally so its unclear as to why things aren't loading properly.

**Comment by totomz** [comment]
Have a look at #45 
TL;DR rename `style.css` in `main.css` as per https://github.com/Pluu/pluu.github.io/commit/d66a9ef6586e7cd207172e683721cd958ed10150 **fix** the issue

**Comment by jaythanelam** [comment]
Hi @totomz! I just made this change and pushed it to Github and I'm not seeing a difference. Got any other ideas?

Thank you ahead of time for trying to help me out. I'm a bit lost as to why this is happening.

**Comment by jaythanelam** [comment]
Since this change wasn't affective should I revert back to the commit prior to this one?

**Comment by jaythanelam** [comment]
@totomz: the site seems to working perfectly now, so thank you for your help. I'm still unclear as to why this change would work. Perhaps if you have the time I'd love an explanation. Either way, thank you again for your help.
